File: backend-laravel/app/Models/Flashcard.php
<?php

namespace App\Models;

use Illuminate\Database\Eloquent\Factories\HasFactory;
use Illuminate\Database\Eloquent\Model;

class Flashcard extends Model
{
    use HasFactory;
    public $timestamps = false;
    protected $fillable = [
        'category_id',
        'question',
        'question_type_id',
        'question_image',
        'answer',
        'answer_type_id',
        'answer_image'
    ];

 
}

File: src/main/java/com/flashcards/FlashCards/DB/Model/Flashcard.java
package com.flashcards.FlashCards.DB.Model;

import lombok.Getter;
import lombok.Setter;

@Getter @Setter
public class Flashcard {
    int id = -1;
    String categoryName="";
    String categoryPath="";
    String question="";
    String answer="";
}
